src/Modules/Power.asm

```diff
@@ -35,9 +35,16 @@
 ;
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 switch_power_off:
+    ; This three macros disable MOSFET communtation
+    ; By them any dshot frame won't have effect until commutation starts
```

Comment:
This comment is a bit unclear, maybe we could re-phrase:
> Dshot frames will be ignored after commutation is started again at motor_start_bidir_done

Reply:
Updated wording